### ISSUE TITLE
Update Rules

### ISSUE THREAD

**Issue by zutrinken** [criticalmaps/criticalmaps-android#254]
<table>
<tr>
<th>current</th>
<th>new (dark)</th>
<th>new (light)</th>
</tr>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/888679/56265866-1062c400-60eb-11e9-9cb0-1706613f0b0b.png" width="240" />
</td>
<td>
<img src="https://user-images.githubusercontent.com/888679/56265867-10fb5a80-60eb-11e9-9a7f-d8d2d75b73cd.png" width="240" /><br>
<img src="https://user-images.githubusercontent.com/888679/56265865-1062c400-60eb-11e9-971e-d938b33a6bc1.png" width="240" />
</td>
<td>
<img src="https://user-images.githubusercontent.com/888679/56265869-10fb5a80-60eb-11e9-91bb-73ee7dc7a4df.png" width="240" /><br>
<img src="https://user-images.githubusercontent.com/888679/56265864-1062c400-60eb-11e9-9d5e-917bdb81109c.png" width="240" />
</td>
</tr>
</table>

**Comment by cbalster** [comment]
Should we wait with this until we have new illustrations?
To be honest a plain list looks a little bland to me. Maybe a Nx2 grid of expandable cards with a nice animation might be an option? (https://material.io/design/components/cards.html)

**Comment by genevievecory** [comment]
Are the current illustrations really by CM Hamburg as the note at the bottom says? The linked domain is for sale: http://criticalmass-hh.de/ - CM Malaga has translated the rules into Spanish and they wanted to credit the source correctly.

**Comment by cbalster** [comment]
Yes, I think the original attribution is - or rather was - correct. The
illustrations have been in the app since around 2014. I would assume the
url quoted has been given up at some point. As to how properly attribute it
I would assume just omitting the url and simply stating "Criticalmass HH"
should be okay?
Message ID: ***@***.***
com>

>
>

**Comment by genevievecory** [comment]
Fine with me!
